core/style.js

```diff
@@ -2117,3 +2117,81 @@ CKEDITOR.tools.extend( CKEDITOR.editor.prototype, {
  * @cfg {String/Array/Boolean} [stylesSet='default']
  * @member CKEDITOR.config
  */
+
+/**
+ * Abstract class describing definition of a style.
```

Comment:
There should be an information how we treat "abstract" classes like that. Generally just put them for information purpose, while they are not existing in the API.

It's important because otherwise one might be trying to instantiate it.

See how other members do that:

* Balloon Panel: https://github.com/ckeditor/ckeditor-dev/blob/5a7a0f441778413a520ab7deead8ce9aebcbb838/plugins/balloonpanel/plugin.js#L834-L835
* Dialog: https://github.com/ckeditor/ckeditor-dev/blob/5a7a0f441778413a520ab7deead8ce9aebcbb838/plugins/dialog/dialogDefinition.js#L15-L16